tomcat_speech/train_and_test_models/test_multitask.py

```diff
@@ -0,0 +1,310 @@
+# train the models created in models directory with MUStARD data
+# currently the main entry point into the system
+import sys
+from datetime import date
+
+import numpy as np
+import copy
+
+# sys.path.append("/net/kate/storage/work/bsharp/github/asist-speech")
+sys.path.append("/work/johnculnan/github/asist-speech")
```

Comment:
@jmculnan It would be best not to have user-specific paths in the code. If you have a virtual environment active and run `pip install -e .` at the root directory of the repo, then in the future whenever that virtual environment is active, it will know about the tomcat_speech package without you having to do `sys.path.append(...`.

Reply:
@adarshp I updated the train and test files to remove these absolute paths.

Reply:
Thanks!